dashboard/config/initializers/fast_localization.rb

```diff
@@ -1,7 +1,7 @@
 # Because loading YAML locales is super-slow, only load english yml locale files in development
 # To load all locales for testing, add "load_locales: true" to locals.yml config
 
-if Rails.env.development? && (!CDO.load_locales)
+if (CDO.skip_locales || Rails.env.development?) && (!CDO.load_locales)
```

Comment:
Did you intend this change to be a part of this PR?  (No objections if you do, if it makes things faster!)